libs/libetpan/src/data-types/mailstream_ssl.c

```diff
@@ -523,6 +544,12 @@ static struct mailstream_ssl_data * ssl_data_new_full(int fd, time_t timeout,
 static struct mailstream_ssl_data * ssl_data_new(int fd, time_t timeout,
 	void (* callback)(struct mailstream_ssl_context * ssl_context, void * cb_data), void * cb_data)
 {
+#if (OPENSSL_VERSION_NUMBER >= 0x10100000L)
+  printf("ssl_data_new TLS_client_method()\n");
+#else
+  printf("ssl_data_new SSLv23_client_method()\n");
```

Comment:
We end up here, which means we will call `ssl_data_new_full()` below with `SSLv23_client_method()`.